cinema/serializers.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
```

Comment:
Consider using `ModelSerializer`, its default implementation of `create` and `update` should satisfy the requirements.